mesheryctl/internal/cli/root/system/start.go

```diff
@@ -105,13 +105,18 @@ func start() error {
 		return err
 	}
 
-	if utils.PlatformFlag == "docker" || utils.PlatformFlag == "kubernetes" {
-		currCtx.SetPlatform(utils.PlatformFlag)
-	} else {
-		if utils.CfgFile != utils.DefaultConfigPath {
+	if utils.PlatformFlag == "" {
+		utils.PlatformFlag = currCtx.GetPlatform()
+	}
```

Comment:
this is not required now. as we are already checking if the platform flag is empty or not ahead .